src/Lykke.Job.HistoryExportBuilder/Startup.cs

```diff
@@ -0,0 +1,225 @@
+                var appSettings = Configuration.LoadSettings<AppSettings>();
+                if (appSettings.CurrentValue.MonitoringServiceClient != null)
+                    _monitoringServiceUrl = appSettings.CurrentValue.MonitoringServiceClient.MonitoringServiceUrl;
+
+                Log = CreateLogWithSlack(services, appSettings);
+
+                builder.RegisterModule(new JobModule(appSettings.CurrentValue.HistoryExportBuilderJob, appSettings.Nested(x => x.HistoryExportBuilderJob), Log));
+                builder.RegisterModule(new ClientModule(appSettings.CurrentValue, Log));
+                builder.RegisterModule(new CqrsModule(appSettings.CurrentValue, Log));
+                
+                builder.Populate(services);
+
+                ApplicationContainer = builder.Build();
+
+                return new AutofacServiceProvider(ApplicationContainer);
+            }
+            catch (Exception ex)
+            {
+                Log?.WriteFatalError(nameof(Startup), nameof(ConfigureServices), ex);
+                throw;
+            }
+        }
+
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplicationLifetime appLifetime)
+        {
+            try
+            {
+                if (env.IsDevelopment())
+                {
+                    app.UseDeveloperExceptionPage();
+                }
+
+                app.UseLykkeForwardedHeaders();
+                app.UseLykkeMiddleware("HistoryExportBuilder", ex => new ErrorResponse {ErrorMessage = "Technical problem"});
+
+
+                app.UseMvc();
+                app.UseSwagger(c =>
+                {
+                    c.PreSerializeFilters.Add((swagger, httpReq) => swagger.Host = httpReq.Host.Value);
+                });
+                app.UseSwaggerUI(x =>
+                {
+                    x.RoutePrefix = "swagger/ui";
+                    x.SwaggerEndpoint("/swagger/v1/swagger.json", "v1");
+                });
+                app.UseStaticFiles();
+
+                appLifetime.ApplicationStarted.Register(() => StartApplication().GetAwaiter().GetResult());
+                appLifetime.ApplicationStopping.Register(() => StopApplication().GetAwaiter().GetResult());
+                appLifetime.ApplicationStopped.Register(() => CleanUp().GetAwaiter().GetResult());
+            }
+            catch (Exception ex)
+            {
+                Log?.WriteFatalError(nameof(Startup), nameof(Configure), ex);
+                throw;
+            }
+        }
+
+        private async Task StartApplication()
+        {
+            try
+            {
+                // NOTE: Job not yet recieve and process IsAlive requests here
+
+                await ApplicationContainer.Resolve<IStartupManager>().StartAsync();
+                await Log.WriteMonitorAsync("", Program.EnvInfo, "Started");
+
+                await AutoRegistrationInMonitoring.RegisterAsync(Configuration, _monitoringServiceUrl, Log);
```

Comment:
Put auto-registration in omnitoring in
#if !DEBUG
....
#endif
to avoid registration on local run